odl/solvers/iterative/statistical.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Maximum Likelihood Expectation Maximization algorithm."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import numpy as np
+
+__all__ = ('mlem', 'loglikelihood')
+
+
+AVAILABLE_MLEM_NOISE = ('poisson',)
+
+
+def mlem(op, x, rhs, niter=1, noise='poisson', callback=None):
+
+    """Maximum Likelihood Expectation Maximation algorithm.
+
+    Attempts to solve::
+
+        min_x L(x | rhs)
+
+    where ``L(x, | rhs)`` is the likelihood of ``x`` given data ``rhs``. The
+    likelihood depends on the underlying noise model and the forward operator.
+
+    Parameters
+    ----------
+    op : `Operator`
+        Operator in the inverse problem. It has to have adjoint that is called
+        via `op.adjoint`
```

Comment:
Huh? First time an operator is mentioned.
So the problem at hand should be stated in short above and perhaps a bit more extensively in the `Notes` below. Also the mention of `op.adjoint` is unnecessary.